### ISSUE TITLE
How to clear a number input field

### ISSUE THREAD

**Issue by mortelee** [microsoft/playwright#2369]
Using playwright 1.0.2

How can we clear an number input field?

We tried using await this.page.fill(selector, ''); but that results in an error that text cannot be entered in a number field. I see in the playwright tests that this is expected behavior.
That leaves the question how we can clear a number input field.

Regards
Marco Ortelee

**Comment by mxschmitt** [comment]
You could use the keyboard to interact with the page to select the text and remove it.

Example:

```js
const playwright = require("playwright");

async function clear(page, selector) {
  const searchInput = await page.$(selector)
  if (!searchInput) {
    throw new Error(`element ${selector} could not be found`)
  }
  await searchInput.click({ clickCount: 3 })
  await searchInput.press('Backspace')
}

(async () => {
  const browser = await playwright.chromium.launch();
  const context = await browser.newContext();
  const page = await context.newPage();
  await page.setContent(`<input type="number" />`)
  await page.type("input", "42")
  await page.screenshot({ path: `1.png` });
  await clear(page, "input")
  await page.screenshot({ path: `2.png` });
  await browser.close();
})();
```

Interactive: https://try.playwright.tech/?s=naq7n

Maybe it makes sense to put such a method into the standard library.

**Comment by dgozman** [comment]
This behavior was accidental, fixed in #2376. Thank you for filing!

**Comment by mortelee** [comment]
Thanks for the quick fix. I really appreciate your fast feedback and commitment.
When can we expect the next release with this fix?
